CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing Guidelines
+
+First, thank you for using VGG! We welcome for issues, pull requests, or asking/answering questions on [Discard channel](#contact-us). Before that, please read [Code of Conduct](CODE_OF_CONDUCT.md).
```

Comment:
grammar problem. `... on Discord channel` should be `... in Discord channel`

Reply:
rejected